3DeT_VICTORY/sheet.json

```diff
@@ -0,0 +1,8 @@
+{
+  "html": "3detvictory.html",
+  "css": "3detvictory.css",
+  "authors": "Bruno Cesar Alves de Freitas Ferreira Mendes",
+  "preview": "3detvictory.png",
```

Comment:
It seems that the name doesn't match any file you have.

Reply:
Added